certified-connectors/Plumsail Documents/apiDefinition.swagger.json

```diff
@@ -1,57 +1,37 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
     "title": "Plumsail Documents",
-    "description": "Generate documents from DOCX, XLSX, PPTX, HTML templates, and populate fillable PDF forms. Automate your contracts, invoices, reports, tickets, and more. Find more information at site https://plumsail.com/documents",
+    "description": "Plumsail Documents connector allows you to set up automated document generation and conversion. The connector also provides a wide range of actions to manipulate PDFs and PDF forms. Create documents from templates, convert to PDF,  split and merge PDF files, protect them, read PDF forms, and more - all you need to automate documents in one connector. Find more information at https://plumsail.com/documents.",
```

Comment:
@RFlipper please make sure that everything is grammatically correct and is free of typographical errors.